examples/models/llama2/CMakeLists.txt

```diff
@@ -121,6 +121,11 @@ if(EXECUTORCH_BUILD_KERNELS_CUSTOM)
   list(APPEND link_libraries custom_ops)
 endif()
 
+if(EXECUTORCH_BUILD_TORCHAO)
+  list(APPEND link_libraries "$<LINK_LIBRARY:WHOLE_ARCHIVE,${CMAKE_CURRENT_BINARY_DIR}/../../../lib/libtorchao_ops_executorch.a>")
```

Comment:
Exporting targets with config in torchao makes this bit nicer.

Reply:
You already called `add_subdirectory` so here you can do something like this:

```
target_link_options(torchao_ops_executorch INTERFACE -Wl,--whole-archive -Wl,--no-whole-archive)

```

Or actually this line should live in torchao/experimental/CMakeLists.txt.